tests/integration/source_code/solacc.py

```diff
@@ -44,16 +45,27 @@ def clone_all():
         run_command(f'git clone {url} {dst}')
 
 
-def lint_all():
+def lint_all(file_to_lint: str | None):
```

Comment:
I'd be tempted to allow a list of files to be passed in here.

Reply:
that doesn't correspond to a usage scenario i.e. either you're checking the entire data set or you're troubleshooting 1 file. But feel free to improve later if you fond it useful.